### ISSUE TITLE
Add `issue` command to show a specific issue publically in the channel

### ISSUE THREAD

**Issue by gkatsev** [smallwins/bugbot#44]
It would be nice to be able to see the details of a particular issue via some command, `issue` for example:
```
/bugbot issue #1234
```
to show details -- the title and the original issue text -- about that specific issue. It should optionally be able to be public or private.

**Comment by brianleroux** [comment]
Love it 👍

**Comment by nickbouton** [comment]
+1

**Comment by RomanMinkin** [comment]
Yeah, that is the missing part for sure. Love this project btw and hope it's not abandoned ;)

**Comment by brianleroux** [comment]
No! Updates forthcoming. HINT https://begin.com is launching sooooooooooon

**Comment by nickbouton** [comment]
@brianleroux where's the poop emoticon when you need it?
